mock_vm/src/contract_register.rs

```diff
@@ -22,16 +22,7 @@ impl ContractRegister {
     ) -> Result<Option<Bytes>, OdraError> {
         let contract = self.contracts.get(addr);
         match contract {
-            Some(container) => {
-                let maybe_error = std::panic::catch_unwind( || {
-                    container.call(entrypoint, args)
-                });
-                if maybe_error.is_err() {
-                    Err(OdraError::VmError(VmError::Panic))
-                } else {
-                    maybe_error.unwrap()
-                }
-            },
+            Some(container) => std::panic::catch_unwind(|| container.call(entrypoint, args))?,
```

Comment:
Why is this logic changed?

Reply:
The logic remains the same, just the notation is more compact.